### PR TITLE
#4874 [Manager]Add a new worker plugin. After restarting the workspac…

### PR DESCRIPTION
**Pull request by wu87988622**
…e successfully, the status of the workspace is not updated

### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4874 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by konekoya** [comment]
Hmm... I cannot seem to restart the workspace 🤔 the dialog won't show up

**Comment by wu87988622** [comment]
> Hmm... I cannot seem to restart the workspace 🤔 the dialog won't show up

new pr are merge maybe is then, but i will test again

File: ohara-manager/client/src/store/epics/workspace/restartWorkspaceEpic.js [hunk 1]
```diff
@@ -487,7 +499,15 @@ export default (action$, state$) =>
         startWorker$(workerKey, skipList),
         waitStartWorker$(workerKey),
 
-        updateWorkspace$(workspaceKey, isRollback),
+        updateWorkspace$(
+          workspaceKey,
+          {
+            worker: state(workerKey, schema.worker),
```
Comment (eechih) [inline ohara-manager/client/src/store/epics/workspace/restartWorkspaceEpic.js:505]: Please use workerSelector.getWorkerById () instead

Comment (wu87988622) [reply ohara-manager/client/src/store/epics/workspace/restartWorkspaceEpic.js:505]: ok

**Comment by wu87988622** [comment]
retry manager-e2e

File: ohara-manager/client/src/store/epics/workspace/restartWorkspaceEpic.js [hunk 2]
```diff
@@ -487,7 +502,18 @@ export default (action$, state$) =>
         startWorker$(workerKey, skipList),
         waitStartWorker$(workerKey),
 
-        updateWorkspace$(workspaceKey, isRollback),
+        updateWorkspace$(
+          workspaceKey,
+          {
+            worker: state(workerSelector.getWorkerById, schema.worker),
```
Comment (eechih) [inline ohara-manager/client/src/store/epics/workspace/restartWorkspaceEpic.js:508]:
```suggestion
            worker: selectors.getWorkerById(state$.value, { id: workerId }),
```

Comment (wu87988622) [reply ohara-manager/client/src/store/epics/workspace/restartWorkspaceEpic.js]: thanks for help

**Comment by wu87988622** [comment]
retry manager-e2e

**Pull request merged**